.github/workflows/main.yml

```diff
@@ -120,18 +120,30 @@ jobs:
         RUST_BACKTRACE: 1
 
   # Install wasm32-unknown-emscripten target, and ensure `crates/wasi-common`
-  # compiles to Emscripten.
-  # TODO enable once rust-lang/rust#66308 is fixed
-  # emscripten:
-  #   name: Emscripten
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #   - uses: actions/checkout@v1
-  #     with:
-  #       submodules: true
-  #   - uses: ./.github/actions/install-rust
-  #   - run: rustup target add wasm32-unknown-emscripten
-  #   - run: cargo build --target wasm32-unknown-emscripten -p wasi-common
+  # compiles to it.
+  emscripten:
+    name: Emscripten
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+    - run: rustup target add wasm32-unknown-emscripten
+    - run: cargo check --target wasm32-unknown-emscripten -p wasi-common
+  
+  # Install armv7-unknown-linux-gnueabihf target, and ensure `crates/wasi-common`
+  # compiles to it.
+  armv7:
+    name: armv7
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+    - run: rustup target add armv7-unknown-linux-gnueabihf
+    - run: cargo check --target armv7-unknown-linux-gnueabihf -p wasi-common
```

Comment:
Since this likely executes pretty quickly I don't think we necessarily need to have two whole builders for this, could these be collapsed perhaps into a single "check all the targets" builder?